lib/integration/notifyConnector.ts

```diff
@@ -48,3 +49,50 @@ export const getNotifyInstance = () => {
 
   return notifyInstance;
 };
+
+export const sendEmail = async (email: string, personalisation: Record<string, string>) => {
+  try {
+    const templateId = process.env.TEMPLATE_ID;
+    if (!templateId && process.env.APP_ENV !== "test") {
+      throw new Error("No Notify template ID configured.");
+    }
+
+    const notify = getNotifyInstance();
+    await notify.sendEmail(templateId, email, { personalisation });
+  } catch (error) {
+    let errorMessage = "";
+
+    if (error instanceof AxiosError) {
+      if (error.response) {
+        /*
+         * The request was made and the server responded with a
+         * status code that falls out of the range of 2xx
+         */
+        const notifyErrors = Array.isArray(error.response.data.errors)
```

Comment:
do we need to check for presence of  data, errors and status?

Reply:
According to the AxiosError type:
```js
export interface AxiosResponse<T = any, D = any> {
  data: T;
  status: number;
  statusText: string;
  headers: RawAxiosResponseHeaders | AxiosResponseHeaders;
  config: InternalAxiosRequestConfig<D>;
  request?: any;
}
```
`status` and `data` will aways be defined on `response`.
If `errors` is not defined than that `undefined` value will filter through and show up as `"... returned the following detailed errors ."`